benchmark-service/Dockerfile

```diff
@@ -11,6 +11,10 @@ RUN pip3 install plotly
 
 RUN pip3 install pandas
 
+RUN pip3 install scipy
+
```

Comment:
It's possible these are already in the base image. Did you try them out?

Reply:
it got mad when I didn't have those lines, not sure if there's another way around it

Reply:
oh, i was thinking of a different docker image. You're right, you need to install it here.